lib/player.rb

```diff
@@ -0,0 +1,28 @@
+class Player
+  attr_reader :token
+
+  def initialize(token)
+    @token = token
+  end
+
+  def move(board)
+    puts "Where do you want to move? Please choose a number 1-9."
+    input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```